feat(workspace): add new resource for app rule restriction (#7386)
huaweicloud/terraform-provider-huaweicloud: docs/resources/workspace_app_rule_restriction.md

---
subcategory: "Workspace"
layout: "huaweicloud"
page_title: "HuaweiCloud: huaweicloud_workspace_app_rule_restriction"
description: |-
  Manages a Workspace restricted application rule resource within HuaweiCloud.
---

# huaweicloud_workspace_app_rule_restriction

> Manages a Workspace restricted application rule resource within HuaweiCloud.

Reviewer: ```suggestion
Manages a Workspace restriction resource to restrict application rules within HuaweiCloud.
```

## Example Usage

```hcl
variable "rule_ids" {
  type: list(string)
}

resource "huaweicloud_workspace_app_rule_restriction" "test" {
  rule_ids = var.rule_ids
}
```

## Argument Reference

The following arguments are supported:

> * `region` - (Optional, String, ForceNew) Specifies the region where the app restricted rule is located.  

Reviewer: ```suggestion
* `region` - (Optional, String, ForceNew) Specifies the region where the restricted application rules are located.  
```
